tools/releaseBuild/azureDevOps/releasePipeline.yml

```diff
@@ -242,7 +249,7 @@ stages:
   displayName: Publish MSIX to store
   jobs:
   - deployment: PublishMsix
-    displayName: Publish MSIX to store
+    displayName: Publish the MSIX Bundle package to store
```

Comment:
This PR only creates the msixbundle, to publish to the Store, we need to use https://www.powershellgallery.com/packages/StoreBroker module.

Reply:
It would be nice to automate the publishing step as well. What is the publish workflow? (what commands to run, what credential is needed?)

Reply:
We can do the publishing in separate PR after we have a release build that produces msixbundle